<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainLoja</class>
 <widget class="QMainWindow" name="MainLoja">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Estoque em Loja</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/png-transparent-minecraft-twitch-video-game-computer-servers-treasure-chest-angle-rectangle-video-game.png</normaloff>../../../Downloads/png-transparent-minecraft-twitch-video-game-computer-servers-treasure-chest-angle-rectangle-video-game.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="CabecLIV">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>LIVROS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="TabLivros">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="CabecCD">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>CDS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="TabCds">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>35</number>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="CabecDVD">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>DVDS</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="TabDVDs">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionLer"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Item</string>
    </property>
    <addaction name="actionIncluir_Livro"/>
    <addaction name="actionIncluir_CD"/>
    <addaction name="actionIncluir_DVD"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuItem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionLer">
   <property name="text">
    <string>Ler...</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar...</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionIncluir_Livro">
   <property name="text">
    <string>Incluir Livro...</string>
   </property>
  </action>
  <action name="actionIncluir_CD">
   <property name="text">
    <string>Incluir CD...</string>
   </property>
  </action>
  <action name="actionIncluir_DVD">
   <property name="text">
    <string>Incluir DVD...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
